src/main/flight/pid.h

```diff
@@ -58,10 +58,10 @@
 
 #define ITERM_ACCELERATOR_GAIN_OFF 0
 #define ITERM_ACCELERATOR_GAIN_MAX 30000
-#define PID_ROLL_DEFAULT  { 42, 85, 35, 90 }
-#define PID_PITCH_DEFAULT { 46, 90, 38, 95 }
-#define PID_YAW_DEFAULT   { 45, 90,  0, 90 }
-#define D_MIN_DEFAULT     { 23, 25, 0 }
+#define PID_ROLL_DEFAULT  { 45, 70, 40, 110 }
```

Comment:
This seems to be containing changes that address different areas (simplified tuning / defaults) - can these please be separated into separate pull requests?

Reply:
Unfortunately it's difficult to not put all this stuff into one PR since it all kind of 'goes together'.
The intent is that sliders will be on by default in 4.3.  See #10798.
This means that the first presentation of the PID numbers to the user won't necessarily be the numbers in the simple 'default' values in those defines above.  They will only see those numbers if all the sliders are centred by default, which may not be the case with the release firmware.  
For example, If we feel that a pitch:roll ratio of 1:1 should happen when the slider is in the middle, which makes a lot of logical sense, then to provide defaults with greater P or D on pitch than roll, the default position of those sliders would be offset right to favour pitch, and the numbers would not look like what we see above.
Likewise the default values for Dmax over Dmin, at default, depend on what 'gain' we have at the centre of the Dmax slider.
This is awkward, and I accept your criticism of changing the defaults at the same time.  I agree that it would be better to change the values in a separate PR.
However for the foregoing reasons I think we just have to do it within this PR, and evaluate the package as a whole.  I thought about this at length and don't see any other simple way.

Reply:
Ah, I see your point - your aim seems to be to provide PID defaults that are derived from the simplified tuning / slider settings. I think this makes sense, as the simplified settings are what we are using to provide simple but flyable settings.
In order to achieve this without having to update the individual PID parameters, and having to do this every time we change how the simplified settings work, we could change CLI `defaults` and `MSP_RESET_CONF` to call `applySimplifiedTuning()` after resetting the settings and before loading the custom defaults. But we can probably first merge this pull request, and then add this later on.

Reply:
Yes, we have a way to go yet before this is working 'as a conceptual demo'... :-(